### ISSUE TITLE
Strange error on second update command

### ISSUE THREAD

**Issue by M1V4D1M** [MEDIARITHMICS/liquibase-clickhouse#16]
Hello everyone! Such a question set up migrations and completed them successfully. Next, I added a new sql file with a new table, I try to start the update and I get this error.

```
Caused by: java.sql.SQLException: no column shaded.liquibase.comment in columns list database table name type default_type default_expression comment
        at shaded.liquibase.ru.yandex.clickhouse.response.ClickHouseResultSet.findColumn(ClickHouseResultSet.java:726)
        at shaded.liquibase.ru.yandex.clickhouse.response.ClickHouseResultSet.getString(ClickHouseResultSet.java:291)
        at shaded.liquibase.ru.yandex.clickhouse.ClickHouseDatabaseMetadata.getColumns(ClickHouseDatabaseMetadata.java:860)
        at liquibase.snapshot.JdbcDatabaseSnapshot$CachingDatabaseMetaData$GetColumnResultSetCache.fastFetchQuery(JdbcDatabaseSnapshot.java:370)
        at liquibase.snapshot.ResultSetCache$SingleResultSetExtractor.fastFetch(ResultSetCache.java:367)
        at liquibase.snapshot.ResultSetCache.get(ResultSetCache.java:72)
```

To be honest, I don't even know where to look.

I`m using liquibase as docker image

```
docker run -it --rm \
  --network=clickhouse-network\
  -v $(pwd)/liquibase/changelog:/liquibase/changelog\
  -v $(pwd)/liquibase/liquibase-clickhouse/target/liquibase-clickhouse-0.7.0-SNAPSHOT-shaded.jar:/liquibase/lib/clickhouse.jar\
  liquibase/liquibase:4.6.1\
  --defaultsFile=/liquibase/changelog/liquibase.properties\
  update
```

With that config:

```
changeLogFile=root-changelog.xml
classpath=/liquibase/changelog
url=jdbc:clickhouse://clickhouse01:8123
username=default
password=
driver=shaded.liquibase.ru.yandex.clickhouse.ClickHouseDriver
databaseClass=liquibase.ext.clickhouse.database.ClickHouseDatabase
liquibase.hub.mode=off
liquibase.logLevel=warning
```

and my root-changelog is

```
<?xml version="1.0" encoding="UTF-8"?>
<databaseChangeLog
        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.6.xsd">
    <includeAll path="sql/v0.1.0"/>
    <includeAll path="sql/v0.2.0"/>
</databaseChangeLog>
```

**Comment by M1V4D1M** [comment]
Hah, I deleted new sql-flie (with new table) and tried to rerun update - and i got same error

**Comment by HomeOfTheWizard** [comment]
Hello,
I have exactly the same error on the second update :(
Actually any command after the first update is having this error. A tag, or rollback too for example.

**Comment by movcale** [comment]
Hi, 
The same issue too. Looks like docker specific or shaded.* packaging

**Comment by eliteWalrus4164** [comment]
as @movcale mentionned it, it is an issue with the shade plugin used to generate shaded jar: https://issues.apache.org/jira/browse/MSHADE-156. It is still an open issue (been open for 9 years now) so it is unclear how to proceed. I'm investigating solutions/workarounds

**Comment by eliteWalrus4164** [comment]
This behavior should be fixed in 0.7.2, closing the issue
